### PR TITLE
feat: FileOpenPicker SettingsIdentifier (for Android)

### PR DESCRIPTION
**Pull request by pkar70**
GitHub Issue (If applicable): #5893 (partly)

## PR Type

What kind of change does this PR introduce?
- Bugfix
- Feature

## What is the current behavior?
SettingsIdentifier  is ignored.

## What is the new behavior?
SettingsIdentifier  is used by FileOpenPicker.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information
Copied from #2896.

Internal Issue (If applicable):

### CODE REVIEW

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UWP/Storage/Pickers/FileOpenPicker.Android.cs [hunk 1]
```diff
@@ -79,8 +96,20 @@ private async Task<FilePickerSelectedFilesArray> PickFilesAsync(bool multiple, C
...
+					{
+						ApplicationData.Current.LocalSettings.Values[settingName] = file.Path;
+						wasPath = true;
+					}
+				}
+
+				if(!wasPath)
+                {	// if we have no path in any of files, remove setting - next call to Picker will not have InitialDir
+					ApplicationData.Current.LocalSettings.Values.Remove(settingName);
+				}
```
Comment (jeromelaban) [inline src/Uno.UWP/Storage/Pickers/FileOpenPicker.Android.cs:112]: Can you adjust the indentation?

**Review by jeromelaban — APPROVED** [review]
Is this what the UWP contract does ?

**Comment by pkar70** [comment]
> Is this what the UWP contract does ?
I think answer is yes, because Microsoft docs says: «To give the user a sense of consistency, the file open picker remembers the last location that the user navigated to and will generally start at that location. Normally, this remembered location is shared by all file open pickers used by the app. To remember a different location for different pickers, choose a different SettingsIdentifier for each picker.»

File: src/Uno.UWP/Storage/Pickers/FileOpenPicker.Android.cs (hunk 1 above)
Comment (pkar70) [reply src/Uno.UWP/Storage/Pickers/FileOpenPicker.Android.cs]: On my side it looks good. But I copied leading white spaces from line 112 to line 110, maybe it helps.

**Pull request merged**